openslides_backend/action/relations/calculated_field_handlers_map.py

```diff
@@ -0,0 +1,27 @@
+from collections import defaultdict
+from typing import Dict, List, Type
+
+from ...models.fields import Field
+from ...models.models import Group, Meeting
+from .calculated_field_handler import CalculatedFieldHandler
+from .meeting_user_ids_handler import MeetingUserIdsHandler
+
+handler_to_field_map: Dict[Type[CalculatedFieldHandler], List[Field]] = {
```

Comment:
Please add a docstring or comment here. This map contains all specific handlers. Maybe we can use a register-decorator later so that you do not have to code here and in the specific file (in this case MeetingUserIdsHandler)

Reply:
If we find to use this more often, this will probably be a good idea. For now I think it's fine.